lib/services/services.c

```diff
@@ -309,6 +309,16 @@ services__create_resource_action(const char *name, const char *standard,
 #endif
 #if SUPPORT_SYSTEMD
     } else if (strcasecmp(op->standard, PCMK_RESOURCE_CLASS_SYSTEMD) == 0) {
+        if (pcmk__strcase_any_of(action, PCMK_ACTION_MONITOR, PCMK_ACTION_STATUS, NULL) && (interval_ms > 0)) {
+     	    if (pcmk__str_eq(g_hash_table_lookup(op->params, PCMK_XA_USE_MONITOR_PENDING_TIMEOUT), "true", pcmk__str_casei)) {
+	        if (interval_ms > timeout) {
+	     		    crm_warn("If monitor-pending-timeout is set to true and the interval is longer than the monitor timeout, "
+		                     "two consecutive monitor deactivations will be treated as an error. "
+	                             "We recommend setting a shorter interval to ensure that "
+	                             "multiple consecutive deactivations before the timeout are treated as an error.");
```

Comment:
I don't follow what the issue is here. The interval is always counted from the last completion, not the last initiation.

Reply:
For example, this is the case when extreme settings such as an interval of 3600s and a timeout of 60s are used.
If the interval is performed at a short interval such as 20s, it can be determined that there is a problem if deactivating is returned several times in a row.

If the setting is 3600s and deactivating is returned the Nth time, the N+1st time will be the second deactivating, so I thought that the number of times might not be enough.

However, on reflection, this warning does not seem to be very meaningless, so I will ultimately delete this warning.